pyvista/core/objects.py

```diff
@@ -357,23 +357,126 @@ def get_data_range(self, arr=None, preference='row'):
 
 
 class Texture(_vtk.vtkTexture, DataObject):
-    """A helper class for vtkTextures."""
+    """A helper class for vtkTextures.
 
-    def __init__(self, *args, **kwargs):
+    Parameters
+    ----------
+    uinput : str, vtkImageData, vtkTexture, list, optional
+        Filename, ``vtkImagedata``, ``vtkTexture``, or a list of images to
+        create a cubemap. If a list of images. Must be of the same size and in
+        the following order:
+
+        * -X
+        * -Y
+        * -Z
+        * +X
+        * +Y
+        * +Z
+
+    **kwargs : dict, optional
+        Optional arguments when reading from a file.
+
+    Examples
+    --------
+    Load a texture from file. Files should be "image" or "image-like" files.
+
+    >>> import os
+    >>> import pyvista
+    >>> from pyvista import examples
+    >>> path = examples.download_masonry_texture(load=False)
+    >>> os.path.basename(path)
+    'masonry.bmp'
+    >>> texture = pyvista.Texture(path)
+    >>> texture  # doctest:+SKIP
+    Texture (0x7f3131807fa0)
+      Components:      3
+      Cube Map:        False
+      Dimensions:      256, 256
+
+    Create a texture from a RGBA array. Note how this is colored per "point"
```

Comment:
The example doesn't seem to include alpha, so that would be RGB. And depending on how you read it it might be better as "_an_ RGB" array.

Reply:
Cleaned it up. You're also correct regarding vowel usage:
https://www.grammar.com/a-vs-an-when-to-use